XS✔ ◾ Do you know how to indicate mandatory vs suggested PR changes? (#9493)
SSWConsulting/SSW.Rules.Content: rules/mandatory-vs-suggested-pr-changes/rule.md

---

> seoDescription: TODO

Reviewer: Does this need attention before we publish?

Author: Sorted

type: rule
title: Do you know how to indicate mandatory vs suggested PR changes?
uri: mandatory-vs-suggested-pr-changes
authors:
  - title: Daniel Mackay
    url: https://www.ssw.com.au/people/daniel-mackay
  - title: Matt Goldman

>     url: https://www.ssw.com.au/people/william-liebenberg

Reviewer: ```suggestion
    url: https://www.ssw.com.au/people/goldie
```

created: 2024-10-28T17:00:00.000Z
guid: 6F5AE6FC-F29A-4938-B012-FDE7F669998C
---

> When reviewing another developers pull request, review comments can fall into a mix of categories. Some comments might be required, while others might be optional. This is where the mandatory vs suggested PR changes rule comes in.

Reviewer: ```suggestion
When reviewing another developer's pull request, review comments can fall into a mix of categories. Some changes might be required, while others might be optional. Do you know the best way to tell the requester which is which?
```

<!--endintro-->

## Types of Pull Request Feedback

> When reviewing pull request comments can often be divided into two categories:

Reviewer: ```suggestion
When reviewing pull requests, comments can often be divided into two categories:
```

> * **Change** - you must change this if you want me to approve.

Reviewer: ```suggestion
* **Change** - you must change this if you want me to approve
```
